cmd/tesla-control/commands.go

```diff
@@ -38,6 +59,67 @@ type Command struct {
 	domain           protocol.Domain
 }
 
+func GetDegree(degStr string) (float32, error) {
+	deg, err := strconv.ParseFloat(degStr, 32)
+	if err != nil {
+		return 0.0, err
+	}
+	if deg < -180 || deg > 180 {
+		return 0.0, errors.New("latitude and longitude must both be in the range [-180, 180]")
+	}
+	return float32(deg), nil
+}
+
+func GetDays(days string) (int32, error) {
+	var mask int32
+	for _, d := range strings.Split(days, ",") {
+		if v, ok := dayNamesBitMask[strings.TrimSpace(strings.ToUpper(d))]; ok {
+			mask |= v
+		} else {
+			return 0, fmt.Errorf("unrecognized day name: %v", d)
+		}
+	}
+	return mask, nil
+}
+
+func TimeRange(rangeStr string) (int32, int32, error) {
```

Comment:
This ended up being unused because of the need to set the Enabled fields automatically.